src/widgets/category/options/options-template.tpl

```diff
@@ -4,10 +4,10 @@
   <p class="CDB-Text is-semibold CDB-Size-small u-upperCase js-textInfo">
     <% if (isLocked) { %>
       <%- totalCats %> blocked <button class="CDB-Text CDB-Size-small u-upperCase u-actionTextColor CDB-Widget-link u-lSpace js-unlock">unlock</button>
-    <% } else if (areAllRejected || rejectedCats === totalCats) { %>
+    <% } else if (areAllRejected || rejectedCatsInData === totalCats || (acceptedCatsInData === 0 && acceptedCats > 0)) { %>
```

Comment:
We should check that there is any category accepted but there isn't any in the current data.